pkg/util/progressbar.go

```diff
@@ -45,38 +42,25 @@ type progressBar struct {
 func (cpb *progressBar) TrackProgress(src string, currentSize, totalSize int64, stream io.ReadCloser) io.ReadCloser {
 	cpb.lock.Lock()
 	defer cpb.lock.Unlock()
-
-	if cpb.pool == nil {
-		cpb.pool = pb.NewPool()
-		if err := cpb.pool.Start(); err != nil {
-			glog.Errorf("pool start: %v", err)
-		}
+	if cpb.progress == nil {
+		cpb.progress = pb.New64(totalSize)
 	}
+	tmpl := `{{ string . "filePath" }} {{ bar . "-" "█" (cycle . "↖" "↗" "↘" "↙" ) "." "-"}} {{speed . }} {{percent .}} {{ rtime . " ETA - %s"}} {{ etime . " Elapsed - %s"}}`
```

Comment:
Any chance you could change this to keep the same UI as before? 

```
	p := pb.Full.Start64(totalSize)
	p.Set("prefix", filepath.Base(src+": "))
```

Reply:
Fixed. Just thought of adding some new pattern :P 

This should be ready for merging now I believe.